pandas/tests/series/methods/test_replace.py

```diff
@@ -675,3 +675,18 @@ def test_replace_value_none_dtype_numeric(self, val):
         result = ser.replace(val, None)
         expected = pd.Series([1, None], dtype=object)
         tm.assert_series_equal(result, expected)
+
+    def test_replace_change_dtype_series(self):
```

Comment:
This seems to be the wrong test? Did you use the correct branch?

Reply:
Yeah, I created a new branch for the task but it's using the last commit as the new commit.

Reply:
I think I didn't update my main before creating the branch. It still shouldn't have used the last commit. Not sure how that happened!